internal/cmd/skupper/listener/listener.go

```diff
@@ -1,6 +1,9 @@
 package listener
 
-import "github.com/spf13/cobra"
+import (
+	"github.com/skupperproject/skupper/internal/cmd/skupper/listener/kube"
+	"github.com/spf13/cobra"
+)
 
 func NewCmdListener() *cobra.Command {
 
```

Comment:
Could you complete this data (Short, Long and Example)? https://skupperproject.github.io/refdog/resources/listener.html

Reply:
added